### PR TITLE
Adds parser for Journal of Vascular and Interventional Radiology

### PR DESCRIPTION
**Pull request by lisahamlett**
### What are you adding/fixing?
Adds parser for https://github.com/emory-libraries/ezpaarse-platforms/issues/246

#### Awesome List for Reviewers of Parsers :star::star:
- [ ] fetch new branch to local repo 'git pull'
- [ ] checkout new branch to test 'git checkout --track origin/{nameofbranch}' / make sure you're on correct branch with 'git status'
- [ ] find proxied resource and look through it (this generates entries in the logs).  Try to go through everything in nav bar since this will likely have most of the domain changes.
- [ ] go to the logs on the main proxy server and grep for your username.  Use these lines to then test.
- [ ] check output from parser directory:  'echo 'line from log' | ./parser.js
- [ ] check manifest.json to make sure no information is missing
- [ ] review/approve/merge pull request based on findings

### CODE REVIEW

**Comment by CB987** [comment]
- should 
https://www.jvir.org:443/content/cme
https://www.jvir.org:443/content/sir_supplements 
also
https://www.jvir.org:443/inpress
https://www.jvir.org:443/issues
https://www.jvir.org:443/issues?publicationCode=jvir&journal=jvir&jc=jvir
be parsed as SEARCH?

- and possibly
https://www.jvir.org:443/current
be parsed as TOC?

- there's also https://www.jvir.org:443/pb/assets/raw/Health%20Advance/journals/jvir/JVIR_28_1_Funaki_Kim-1482330661327.mp3
found from going here
https://www.jvir.org:443/multimedia
neither of which parse, but possibly could be SUPPLEMENT(as they are tied to articles) and TOC, respectively?

**Comment by AGCooper** [comment]
Also you'll want to fix line 40.

alex@alex-iMac:~/ezpaarse-platforms/jvir$ echo "https://www.jvir.org:443/article/S1051-0443(16)30566-8/addons" | ./parser.js{"mime":"SUPPLEMENT","rtype":"ARTICLE","unitid":"S1051-0443(16)30566-8"}

And PII is also an rid, as in https://www.jvir.org:443/action/showFullTextImages?pii=S1051-0443%2816%2930566-8

:bowtie:

**Comment by lisahamlett** [comment]
I just realized I missed the mp3.  I shall add that now.

**Comment by lisahamlett** [comment]
And by MR3, I mean MP3

**Comment by AGCooper** [comment]
Check this out https://search.crossref.org/?q=10.1016%2FS1051-0443%2819%2930265-9

File: jvir/parser.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    result.mime   = 'HTML';
+  } else if ((match = /^\/cms\/([0-9.]+)\/([a-zA-Z0-9.]+)\/([a-z]+)\/([a-zA-Z0-9-]+)\/([a-zA-Z0-9.]+)$/i.exec(path)) !== null) {
+    // https://www.jvir.org:443/cms/10.1016/j.jvir.2016.09.028/attachment/58dd870f-e7e3-40e9-afed-1897cd74fe9c/mmc2.mp4
+    // https://www.jvir.org:443/cms/10.1016/j.jvir.2016.09.028/attachment/8f4a8bdc-6d6a-4fb2-a962-2d5691cfd0c6/mmc1.mp4
+    result.rtype  = 'SUPPLEMENT';
+    result.mime   = 'VIDEO';
+    result.doi    = match[1] + '/' + match[2];
+    result.unitid = match[4] + '/' + match[5];
+  } else if ((match = /^\/pb\/assets\/raw\/Health%20Advance\/journals\/jvir\/([a-zA-Z0-9_.-]+)$/i.exec(path)) !== null) {
+    // https://www.jvir.org/pb/assets/raw/Health%20Advance/journals/jvir/JVIR_28_1_Funaki_Kim-1482330661327.mp3
```
Comment (AGCooper) [inline jvir/parser.js:74]: maybe change this to `^\/pb\/assets\/raw\/Health%20Advance\/journals\/jvir\/([a-zA-Z0-9_.-]+).mp3$` to make sure it pnly parses mp3s?

Comment (lisahamlett) [reply jvir/parser.js:74]: Done

**Review by CB987 — APPROVED** [review]

**Pull request merged**